cli/commands/remote/create/table/index.md

```diff
@@ -6,29 +6,27 @@ permalink: /cli/commands/remote/create/table/
 * TOC
 {:toc}
 
-**Create a [table](https://help.keboola.com/storage/tables/) in Keboola Storage.**
+## Create a Table from the CLI
+
```

Comment:
Why is this needed?

![image](https://github.com/keboola/developers-docs/assets/19371734/a424b48a-ccc6-4161-8af6-72c686f032fc)

Reply:
> I automatically did it the same way we do it in the user docs for consistency.

I still don't understand two almost identical headings below each other, I would understand the subheading: `Into` or `Overview`.